Makefile

```
@@ -0,0 +1,80 @@
+.PHONY: build test clean run install fmt vet lint help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)